packages/react-devtools-shared/src/__tests__/dehydratedValueSerializer.js

```diff
@@ -10,10 +10,14 @@
 // test() is part of Jest's serializer API
 export function test(maybeDehydratedValue) {
   const {meta} = require('react-devtools-shared/src/hydration');
+
+  const hasOwnProperty = Object.prototype.hasOwnProperty.bind(
+    maybeDehydratedValue,
+  );
```

Comment:
Can you explain why this was necessary?

Reply:
I think this was specifically for https://github.com/facebook/react/pull/20618/files#diff-11c09cf7ec0ed466fcabb7f3abc798d407062cacc789d56d89f2c56c1584d19fL777-L780. But in general I would consider this good practice in case we receive objects such as `{ hasOwnProperty: null }`

Reply:
Oh right, I forgot about that. Ugh. lol